### PR TITLE
Make WingetUI to UniGetUI

### PR DESCRIPTION
**Pull request by skanda890**
- [X] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding), and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**.
- [X] **Have you checked that there aren't other open [pull requests](https://github.com/marticliment/WingetUI/pulls) for the same changes?**
- [X] **Have you tested that the committed code can be executed without errors?**
- [X] **This PR is not composed of garbage changes used to farm GitHub activity to enter potential Crypto AirDrops.**
Any user suspected of farming GitHub activity with crypto purposes will get banned. Submitting broken code wastes the time of the contributors, who have to spend their free time reviewing, fixing, and testing code that does not even compile, breaks other features, or does not introduce any useful changes. Thank you for your understanding.

-----
This changes README.md and WebBasedData/screenshot-database-v2.json

-----

### CODE REVIEW

**Comment by skanda890** [comment]
Do not close or merge the PR until all changes are done.

**Comment by skanda890** [comment]
Can you mention where we have to change in GitHub.

**Review by marticliment — APPROVED** [review]

**Comment by marticliment** [comment]
Id the PR is not ready, please convert it to draft

**Comment by marticliment** [comment]
> Note: Only merge this when everything which needs to be changed like: paths from marticliment/WingetUI to marticliment/UniGetUI etc.

Please do not change this, I have already all the changes prepared on my local pc, ready to merge

**Comment by skanda890** [comment]
> > Note: Only merge this when everything which needs to be changed like: paths from marticliment/WingetUI to marticliment/UniGetUI etc.
> 
> Please do not change this, I have already all the changes prepared on my local pc, ready to merge

Sure.

**Comment by skanda890** [comment]
How do I change it to draft?

**Comment by skanda890** [comment]
How do I change it to draft?

**Comment by marticliment** [comment]
I have changed it to draft

**Comment by skanda890** [comment]
> I have changed it to a draft.

Okay.

**Review by skanda890 — COMMENTED** [review]
@marticliment, How do you fix these conflicts?
image

**Comment by skanda890** [comment]
@marticliment, can you make this PR ready?

**Comment by skanda890** [comment]
@marticliment, have you changed all the files that are supposed to be changed yet?

**Comment by marticliment** [comment]
No, I have not. I will change them once I publush 3.1.0 stable

**Comment by marticliment** [comment]
> @marticliment, How do you fix these conflicts? image

Please revert the commit updating the screenshot-database file.
Do not update screenshot-database file manually on a branch, since then it causes conflicts to appear

**Comment by skanda890** [comment]
> > @marticliment, How do you fix these conflicts? image
> 
> Please revert the commit updating the screenshot-database file. Do not update screenshot-database file manually on a branch, since then it causes conflicts to appear

I reverted it but still there.

**Comment by marticliment** [comment]
I will undraft when I have migrated all .MD files to UniGetUI

**Comment by skanda890** [comment]
Sure.

**Comment by skanda890** [comment]
Have you updated all the paths?

**Comment by marticliment** [comment]
Which paths?

File: README.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
-# <img src="https://raw.githubusercontent.com/marticliment/WingetUI/main/src/wingetui/Assets/Images/icon.png" height="40">UniGetUI (formerly WingetUI)
+# <img src="https://raw.githubusercontent.com/marticliment/WingetUI/main/media/icon.png" height="40">UniGetUI
```
Comment (skanda890) [inline README.md]: Like this one etc.

Comment (marticliment) [reply README.md]: Yes, in theory they should be all ok

Comment (skanda890) [reply README.md]: > Yes, in theory they should be all ok

Change paths from marticliment/WingetUI to marticliment/UniGetUI

Comment (marticliment) [reply README.md]: Not yet, I have not renamed the repo yet, and don't worry about this, since github will redirect WingetUI to UniGetUI

**Review by marticliment — APPROVED** [review]

**Pull request merged**